archaius2-core/src/main/java/com/netflix/archaius/config/CompositeConfig.java

```diff
@@ -230,6 +230,14 @@ public Config getConfig(String name) {
     }
 
     @Override
+    public void setListDelimiter(String newDelimiter) {
```

Comment:
Also add setListDelimiter to postConfigAdded() which is called for any child that is added.  This will address the case where a child is added after setListDelimiter() is called